app/views/social_previews/articles/shecoded.html.erb

```diff
@@ -1,47 +1,103 @@
 <style>
   body {
-    <% if @article.cached_tag_list.include?("theycoded") %>
-      background-color: #cabde6;
-    <% else %>
-      background: url(<%= asset_path "SheCodedBackground.png" %>);
-      background-color: #cabde6;
-      background-repeat: repeat;
-    <% end %>
+    margin: 0;
+    padding: 0;
+    font-family: "Roboto", "Helvetica Neue", Helvetica, Arial, "Lucida Grande", sans-serif;
+    background: white;
   }
   .preview-div-wrapper {
-    background: white;
-    width: 80vw;
-    height: 35vw;
-    margin: 2.5vw auto 2.5vw;
-    padding: 4vw;
-  }
-  .avatar {
-    margin-left: auto;
-    margin-right: auto;
-    margin-bottom: 2.5vw;
-    width: 12vw;
-  }
-  img {
-    border-radius: 12vw;
-    border: .75vw solid #70589b;
-    height: 12vw;
-    width: 12vw;
-  }
-  .title {
-    font-size: <%= 7 - @article.title.size / 40 %>vw;
-    margin: auto;
-    width: 80%;
-    color: #70589b;
-    text-align: center;
-    font-family: "HelveticaNeue-CondensedBold", "HelveticaNeueBoldCondensed", "HelveticaNeue-Bold-Condensed", "Helvetica Neue Bold Condensed", "HelveticaNeueBold", "HelveticaNeue-Bold", "Helvetica Neue Bold", "HelveticaNeue", "Helvetica Neue", 'TeXGyreHerosCnBold', "Helvetica", "Tahoma", "Geneva", "Arial Narrow", "Arial", "Roboto Condensed", sans-serif;
+    overflow: hidden;
+    position: relative;
+  }
+  .preview-div {
+    border-radius: 25px 25px 0 0;
+    background: url(/assets/shecoded-background.jpg);
+    background-position: 50% 50%;
+    background-size: 100%;
+    overflow: hidden;
+    width: 94vw;
```

Comment:
I can't yet be sure of the width, height and background position in this div, but it's pretty difficult to see the end result without a twitter card (which we cannot preview without a production link - i did try and do it with ngrok). I tried to get it looking as good as possible within the dimensions as per the design.

Reply:
It's difficult to control the doodles on the background